keps/sig-network/20190125-ingress-api-group.md

```diff
@@ -362,6 +372,7 @@ This resource will serve a couple of purposes:
   associated with a given Ingress controller.
 
 Add a field to `ingress.spec`:
+
 ```golang
 type IngressSpec struct {
   ...
```

Comment:
> Class references an IngressClass resource in kube-system

why in `kube-system`? are there other examples of `*Class` resources that apply to the whole cluster but exist in a single namespace? what is the meaning of two IngressClass resources with the same name in different namespaces?

Reply:
I really don’t like kube-system being special or hardcoded for non core code.

Reply:
Made IngressClass a cluster level resource